elements/pl-checkbox/pl-checkbox.py

```diff
@@ -15,7 +15,7 @@ def prepare(element_html, element_index, data):
     incorrect_answers = []
     index = 0
     for child in element:
-        if child.tag == 'pl_answer':
+        if child.tag in ['pl-answer', 'pl_answer']:
```

Comment:
I think you are writing python :-) I believe JS needs [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes), not [in](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in) for arrays.

Reply:
This is python though, right? I'm confused 😂

Reply:
Hahahaha! I was the most confused of all :-)